mmdet/datasets/pipelines/loading.py

```diff
@@ -32,16 +32,20 @@ class LoadImageFromFile:
         file_client_args (dict): Arguments to instantiate a FileClient.
             See :class:`mmcv.fileio.FileClient` for details.
             Defaults to ``dict(backend='disk')``.
+        image_backend (str): The backend argument for :func:`mmcv.imfrombytes`.
```

Comment:
Please modify `LoadMultiChannelImageFromFiles` and related loading classes at the same time.